src/platform/datapath_epoll.c

```diff
@@ -2164,7 +2170,7 @@ CxPlatSocketSendInternal(
     ProcNumber = CxPlatProcCurrentNumber() % Socket->Datapath->ProcCount;
     SocketContext = &Socket->SocketContexts[Socket->HasFixedRemoteAddress ? 0 : ProcNumber];
 
-    ProcContext = &Socket->Datapath->ProcContexts[ProcNumber];
+    ProcContext = &Socket->Datapath->ProcContexts[Socket->HasFixedRemoteAddress ? Socket->ProcIndex : ProcNumber];
```

Comment:
Is `ProcNumber` needed if `HasFixedRemoteAddress`?

Reply:
Technically no. I can switch it to an if on the HasFixedRemoteAddress.